class DeniedError(Exception):
    pass


class UnexpectedMessageError(Exception):
    pass
